<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellerShoppingitemForm</class>
 <widget class="QWidget" name="SellerShoppingitemForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_price">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
    </font>
   </property>
   <property name="text">
    <string>单价</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bookName">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光标题宋_CNKI</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>书名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>单价：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_condition">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>150</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正姚体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>561</x>
     <y>151</y>
     <width>16</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_time">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>220</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_number">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font-weight:800; 
color:rgb(255,0,0);
</string>
   </property>
   <property name="text">
    <string>数量</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bookImage">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>150</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>100</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗黑宋简体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>订单：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>782</x>
     <y>54</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Baskerville Old Face</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_buyerId">
   <property name="geometry">
    <rect>
     <x>858</x>
     <y>51</y>
     <width>111</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light SemiCondensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
